0-ad-history-mod-package/simulation/templates/units/han/ship_trireme.xml

```diff
@@ -8,6 +8,12 @@
     <Civ>han</Civ>
     <SpecificName>Lóu Chuán</SpecificName>
     <Icon>units/han/ship_trireme.png</Icon>
+    <History>
+        The most prevalent ships in Han fleets were so-called tower or castle ships. They came in all different sizes, with the largest ones measuring over 110 feet (33 meters) in height and carrying hundreds of soldiers. The larger ones formed the core of the fleet, while the smaller ones served as support ships. Tower ships are often described as floating fortresses. This is because they had multiple stories of wooden bulwarks looking like buildings standing on deck, protecting the soldiers inside. However, the rowers were not positioned inside the bulwarks. The largest variants comprised three stories, others only one or two.
+        There was one big weakness of this towering design: its high center of gravity resulted in comparably low stability on the water, disabling it from really driving on the high sea. However, this posed no problem since the Han navy mostly operated in coastal or riverine regions anyway.
```

Comment:
```suggestion
        There was one big weakness of this towering design: its high center of gravity resulted in a comparably low stability on the water, disabling it from really sailing on the high seas. However, this posed no problem since the Han navy primarily operated in coastal or riverine regions.
```